tests/test_macros.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(specialization)]
```

Comment:
I don't want to have too many test files.
Could you please move this test to `test_class_new` or so?

Reply:
Hold fire on that thought. I will add in the additional tests and we can agree where to put them once I’ve made sure we can macro_rules create anything with an attribute.